examples/p4est_2d_dgsem/elixir_euler_cylinder_bowshock_mach3.jl

```diff
@@ -0,0 +1,176 @@
+using Trixi
+using OrdinaryDiffEqSSPRK
+
+###############################################################################
+# Geometry & boundary conditions
+
+# Mapping to create a "close-up" mesh around the second quadrant of a cylinder,
+# implemented by Georgii Oblapenko. If you use this in your own work, please cite:
+#
+# - G. Oblapenko and A. Tarnovskiy (2024)
+#   Reproducibility Repository for the paper:
+#   Entropy-stable fluxes for high-order Discontinuous Galerkin simulations of high-enthalpy flows.
+#   [DOI: 10.5281/zenodo.13981615](https://doi.org/10.5281/zenodo.13981615)
+#   [GitHub](https://github.com/knstmrd/paper_ec_trixi_chem)
+#
+# as well as the corresponding paper:
+# - G. Oblapenko and M. Torrilhon (2025)
+#   Entropy-conservative high-order methods for high-enthalpy gas flows.
+#   Computers & Fluids, 2025.
+#   [DOI: 10.1016/j.compfluid.2025.106640](https://doi.org/10.1016/j.compfluid.2025.106640)
+#
+# The mapping produces the following geometry & shock (indicated by the asterisks `* `):
+#                  ____x_neg____
+#                 |             |
+#               |               |
+#             |                 |
+#            |                * |
+#           |               *   y
+#          |   Inflow     *     _
+#         |    state    *       p
+#         x           *         o
+#        _           *          s
+#       n           *           |
+#      e           *            |
+#     g         Shock          . 
+#     |          *           . 
+#    |          *          .  <- x_pos
+#   |          *          .    
+#  |          *         .  (Cylinder)
+#  |_______y_neg_______.
+function mapping_cylinder_shock_fitted(xi_, eta_,
+                                       cylinder_radius, spline_points)
+    shock_shape = [
+        (spline_points[1], 0.0),
+        (spline_points[2], spline_points[2]),
+        (0.0, spline_points[3])
+    ] # 3 points that define the geometry of the mesh which follows the shape of the shock (known a-priori)
+    R = [sqrt(shock_shape[i][1]^2 + shock_shape[i][2]^2) for i in 1:3]  # 3 radii
```

Comment:
Can you please add a comment on how these spline points were obtained and what's their geometric meaning?

Reply:
Thank you for the explanation, @knstmrd, and thanks @DanielDoehring for adding the comments to the elixir!